doc/api/n-api.md

```diff
@@ -105,6 +105,17 @@ opt-in to access those APIs:
 In this case the entire API surface, including any experimental APIs, will be
 available to the module code.
 
+### Badges
+
+The use of badges is recommended to indicate the minimum version of N-API 
+required for the module. This helps to determine which Node.js major versions 
+are supported. The following badges are available:
+
+![N-API v1 Badge](../api_assets/N-API%20v1%20Badge.svg)
+![N-API v2 Badge](../api_assets/N-API%20v2%20Badge.svg)
+![N-API v3 Badge](../api_assets/N-API%20v3%20Badge.svg)
+![N-API Experimental Version Badge](../api_assets/N-API%20Experimental%20Version%20Badge.svg)
```

Comment:
Have you verified that this works in the `make doc` output? I think most of our assets are included via `assets/`, not some `../`-prefixed path